src/sssom/util.py

```diff
@@ -151,6 +157,12 @@ def from_mapping_set_document(cls, doc: MappingSetDocument) -> "MappingSetDataFr
         df = pd.DataFrame(get_dict_from_mapping(mapping) for mapping in doc.mapping_set.mappings)
         meta = _extract_global_metadata(doc)
 
+        if pandas_version >= (2, 0, 0):
```

Comment:
Is there still push-back in the community (linkml etc) to simply depend on >=2.0.0 instead of doing a hack like this?

Reply:
Not sure about the pushback. Since we had tests specific to older pandas versions in our workflow, I added this.

Reply:
hmmmm I would kinda like to consider to drop pandas 1.X support, especially since 3.0 is one the way. Do you know anyone who has an opinion about this?

Reply:
Not sure who raised this issue in the first place. For some reason I feel it was either Charlie or Chris. Do not recollect it at all.

Reply:
Isn't oak already on pandas 2? Thats what Chris said on slack..

Reply:
Oak was never the issue. I think it's #432

Reply:
I am ok with merging this but next time we see ourselves introducing code to handle pandas 1.x I think we should refactor and get rid of the conditionals

Reply:
I agree!